Library/Homebrew/blacklist.rb

```diff
@@ -1,16 +1,5 @@
 def blacklisted? name
   case name.downcase
-  when 'screen', /^rubygems?$/ then <<-EOS.undent
```

Comment:
Oh, I missed your last comment, @xu-cheng.

I would agree, except having these formulae blacklisted prevents Homebrew from informing the user that it does exist in a tap.  Currently:
```
🐙  brew install libpcap
Error: No available formula for libpcap
Apple distributes libpcap with OS X, you can find it in /usr/lib.
```
With this change:
```
🐙  brew install libpcap
Error: No available formula for libpcap
==> Searching formulae...
==> Searching taps...
homebrew/dupes/libpcap
```